src/app/Fake.DotNet.MSBuild/MSBuild.fs

```diff
@@ -65,6 +65,42 @@ type MSBuildDistributedLoggerConfig =
       AssemblyPath : string
       Parameters : (string * string) list option }
 
+#if !NETSTANDARD1_6
```

Comment:
This #ifdef can now dropped, since now we target netstandard2.0 instead of 1.6?

Reply:
👍